internal/services/users/user_data_source.go

```diff
@@ -156,6 +156,11 @@ func userData() *schema.Resource {
 				Computed:    true,
 				Description: "The primary cellular telephone number for the user.",
 			},
+
+			"user_type": {
+				Type:     schema.TypeString,
+				Computed: true,
```

Comment:
The other schema items have the description field, is it worth including here for completeness?

Reply:
Sure, can do 👍